Java/PlatinumGenomes-variant-transformation/src/main/java/com/google/cloud/genomics/examples/TransformNonVariantSegmentData.java

```diff
@@ -153,8 +164,13 @@ private static TableSchema getTableSchema() {
     public void processElement(ProcessContext c) {
       Variant v = c.element();
 
+      HashMultiset genotypeCount = HashMultiset.create();  // This will typically hold counts for genotypes -1,0,1,2.
```

Comment:
You use this variable to extract the count, but the variable itself does not hold counts. (Right?)
If so, I think the variable should be renamed like allVariantGenotypes or some such.